src/main/scala/com/advancedtelematic/campaigner/http/UpdateResource.scala

```diff
@@ -97,8 +97,13 @@ class UpdateResource(extractNamespace: Directive1[Namespace], deviceRegistry: De
   val route: Route =
     extractNamespace { ns =>
       pathPrefix("updates") {
-        (path(UpdateId.Path) & pathEnd) { updateUuid =>
-          complete(updateRepo.findById(updateUuid))
+        path(UpdateId.Path) { updateUuid =>
+          get {
+            complete(updateRepo.findByIdUnsafe(updateUuid))
```

Comment:
I'd like to make this `updateRepo.findById(ns, updateUuid)` but I don't know who's using it and if it might break something (?)..